internal/ui/settings_ui.go

```diff
@@ -105,25 +105,25 @@ func (d *settingsUI) loadAppearanceScreen() fyne.CanvasObject {
 		themeList.Add(themeButton)
 	}
 
-	bg := fyne.NewContainerWithLayout(layout.NewBorderLayout(nil, nil, bgLabel, bgButtons), bgLabel, bgPath, bgButtons)
-	time := fyne.NewContainerWithLayout(layout.NewBorderLayout(nil, nil, clockLabel, clockFormat), clockLabel, clockFormat)
+	bg := container.NewBorder(nil, nil, bgLabel, bgButtons, bgPath)
+	time := container.NewBorder(nil, nil, clockLabel, clockFormat)
 	top := container.NewVBox(bg, time)
 
 	themeFormLabel := widget.NewLabelWithStyle("Icon Theme", fyne.TextAlignCenter, fyne.TextStyle{Bold: true})
-	themeCurrent := widget.NewHBox(layout.NewSpacer(), themeLabel, themeIcons)
-	bottom := fyne.NewContainerWithLayout(layout.NewBorderLayout(nil, themeCurrent, themeFormLabel, nil), themeCurrent, themeFormLabel, container.NewScroll(themeList))
+	themeCurrent := container.NewHBox(layout.NewSpacer(), themeLabel, themeIcons)
+	bottom := container.NewBorder(nil, themeCurrent, themeFormLabel, nil, container.NewScroll(themeList))
 
-	applyButton := widget.NewHBox(layout.NewSpacer(),
```

Comment:
Doesn't removing this spacer break the right alignment?

Reply:
Good catch. That was not intentionally. Will get it sorted tomorrow.